docs/Using-Fleet/Permissions.md

```diff
@@ -124,7 +124,7 @@ Users that are members of multiple teams can be assigned different roles for eac
 | Run any query as [live query](https://fleetdm.com/docs/using-fleet/fleet-ui#run-a-query)                                         |               | ✅             | ✅              | ✅         |             |
 | Create, edit, and delete only **self authored** queries                                                                          |               |                | ✅              | ✅         | ✅          |
 | View all queries\**                                                                                                              | ✅            | ✅             | ✅              | ✅         |             |
-| Add, edit, and remove queries from the schedule                                                                                  |               |                | ✅              | ✅         | ✅          |
+| Manage [query automations](https://fleetdm.com/docs/using-fleet/fleet-ui#schedule-a-query)                                       |               |                 |                 | ✅         |             |
```

Comment:
I think the team maintainer and team GitOps columns still get checkmarks; see previous comment